profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/create-sam-perimeter-resource.sh

```diff
@@ -37,4 +35,12 @@ curl -X POST --header 'Content-Type: application/json' --header "Authorization:
      } \
    }, \
    \"authDomain\": [] \
- }" "${SAM_HOST}/api/resources/v1/service-perimeter"
+ }" "${SAM_HOST}/api/resources/v1/service-perimeter")
+
+# We consider the 200s as success, and also 409 if the resource already exists. If the resource already exists, this
+# script may have already run and that's ok.
+if [[ $STATUS -eq '200' ]] || [[ $STATUS -eq '204' ]] || [[ $STATUS -eq '409' ]]; then
```

Comment:
This endpoint will return a 201 or a 204 depending on whether `returnResource` is true or false, but I don't believe it will ever return 200 so we should change this check to account for that